app/src/main/java/de/ur/explure/views/SingleRouteFragment.kt

```diff
@@ -108,7 +110,7 @@ class SingleRouteFragment : Fragment(R.layout.fragment_single_route), KoinCompon
             // start route
         }
         binding.shareRouteButton.setOnClickListener {
```

Comment:
Wärs vielleicht schöner den Button lieber als Menü-Item oben in die Toolbar zu packen? Dann wird vielleicht auch etwas klarer, dass das ein klickbarer Button ist